system_tests/test_vsphere_local_blueprints/__init__.py

```diff
@@ -82,7 +84,7 @@ def check_vm_name_in_runtime_properties(runtime_props, name_prefix, logger):
     check_name_is_correct(name, name_prefix, logger)
 
 
-def check_correct_vm_name(vms, name_prefix, logger):
+def check_correct_vm_name(vms, name_prefix, logger, hyphens=2):
```

Comment:
...and then I removed that variable from the function later when I realised it made something else harder. Whoops.